src/components/forms/RangeInput/RangeInput.stories.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react'
+import { RangeInput } from './RangeInput'
+
+export default {
+  title: 'Forms/RangeInput',
+  parameters: {
+    info: `
+USWDS 2.0 RangeInput component
+
+Source: https://designsystem.digital.gov/components/form-controls/#range
+`,
+  },
+}
+
+const labelChildren = (
+  <>
+    <div>
+      <span>Start:</span> <span>0</span>
+    </div>
+    <div>
+      <span>End:</span> <span>10</span>
+    </div>
+  </>
+)
+
+const labelHint = <>(Some hint)</>
```

Comment:
it looks like there are a few unused variables now that can be deleted. FWIW I think it would be fine to include a story that renders the `RangeInput` alongside a `Label` if you wanted to demo that.